### DISCUSSION TITLE
does anyone know how to make a vlookup [May 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2085] [Discord Threads]
> 🙋‍♂️ **CaptainKranch**  
## does anyone know how to make a vlookup on another table? I know it has an option to link to another table and then use the lookup function but if I have multiple records do I have to enter manually each link? My work around was to use the API to bulk update the link column but when I use the swagger API it doesn't shows the link column, am I doing something wrong or does anyone knows how to do a VLOOKUP function with another table?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/971498116572397628/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/971498116979257424/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### sorry, vlookup isnt available  
#  
> 🙋‍♂️ **CaptainKranch**  
#### Ok!! thanks for the help  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/971498118359158805)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by mayeulk** [comment]
> > 🙋‍♂️ **CaptainKranch**
> 
> ## does anyone know how to make a vlookup on another table? I know it has an option to link to another table and then use the lookup function 
I guess you refer to the method shown in this video, which might help those doing this manually: https://www.youtube.com/watch?v=YB1I3VpY7yU
